pkg/cmd/logs/http/http.go

```diff
@@ -0,0 +1,110 @@
+package http
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/MakeNowJust/heredoc"
+	msg "github.com/aziontech/azion-cli/messages/logs/http"
+	"github.com/aziontech/azion-cli/pkg/api/graphql/http"
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/aziontech/azion-cli/pkg/logger"
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+)
+
+var (
+	tail      bool
+	pretty    bool
+	startTime = time.Now()
+	utcTime   = startTime.UTC()
+	logTime   time.Time
+	limit     string
+)
+
+func NewCmd(f *cmdutil.Factory) *cobra.Command {
+	logTime = utcTime.Add(-5 * time.Minute)
+	cmd := &cobra.Command{
+		Use:           msg.Usage,
+		Short:         msg.ShortDescription,
+		Long:          msg.LongDescription,
+		SilenceUsage:  true,
+		SilenceErrors: true, Example: heredoc.Doc(`
+		$ azion logs http
+		$ azion logs http --tail
+		`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := printLogs(cmd, f)
+			if err != nil {
+				return err
+			}
+			return nil
+		},
+	}
+
+	cmd.Flags().BoolP("help", "h", false, msg.FlagHelp)
+	cmd.Flags().StringVar(&limit, "limit", "100", msg.LimitFlag)
+	cmd.Flags().BoolVar(&tail, "tail", false, msg.FlagTail)
+	cmd.Flags().BoolVar(&pretty, "pretty", false, msg.FlagPretty)
+	return cmd
+}
+
+func printLogs(cmd *cobra.Command, f *cmdutil.Factory) error {
+
+	resp, err := http.HttpEvents(f, logTime, limit)
+	if err != nil {
+		return err
+	}
+
+	for _, event := range resp.HTTPEvents {
+		if tail && logTime.After(event.Ts) {
+			continue
+		}
+
+		colorLog := color.FgGreen
+
+		if pretty {
+			color.New(colorLog).Fprint(f.IOStreams.Out, "Timestamp: ")
```

Comment:
it would be nice to have a function for all logs

Reply:
Each query has different fields, so it would still require separate work for each query